lib/client-assets.php

```diff
@@ -817,7 +818,7 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	$allowed_block_types = apply_filters( 'allowed_block_types', true );
 
 	$editor_settings = array(
-		'wideImages'       => ! empty( $gutenberg_theme_support[0]['wide-images'] ),
+		'alignWide'        => $align_wide,
```

Comment:
I'm thinking for one Gutenberg release or two, we should support both and maybe warn the user (console.warn for instance) if we detect the usage of the old `gutenberg` theme support?

Reply:
Sure, can do. Wasn't sure how much backcompat is expected at the moment.

Reply:
Yeah! not sure either, but keeping it for one or two releases seems reasonable to me. We're doing the same thing (warning) for the deprecated block attributes definition.

Reply:
I love the idea of supporting both for a few releases and maybe throwing a `_doing_it_wrong` or similar for `wideImages`